README.md

```diff
@@ -10,36 +10,36 @@ different criteria such as scope, author, year, etc.
 Requirements
 ------------
 
-* Django 1.10 - 2.0
-* Python 2.7, 3.4 - 3.6
+* Django 1.11, 2.0
+* Python 2.7, 3.5- 3.7
 
 
 Installation
 ------------
 
 1.  Download and install from source code.
     ```sh
-	$ pip install git+git://github.com/unt-libraries/django-citeit.git
+	$  pip install git+git://github.com/unt-libraries/django-citeit.git
     ```
 
 2.  Add app to INSTALLED_APPS.
     ```python
-	INSTALLED_APPS = (
+	   INSTALLED_APPS = (
             'citeIt',
-        )
+       )
     ```
 
 3.  Include the URLs.
     ```python
-	urlpatterns = [
+	    urlpatterns = [
             url(r'^admin/', admin.site.urls),
             url(r'^withers/', include('citeIt.urls'))
         ]
     ```
 
 4.  Migrate/sync the database
     ```sh
-	$ python manage.py migrate
+	$  python manage.py migrate
```

Comment:
I don't think any of these spacing changes are needed. It makes the spacing inconsistent in raw form and with what we usually do, and we are not using the same IDEs, so basing spacing on a specific IDE is not ideal.

Reply:
Ummm...ok. If it is working fine in all your IDEs except just mine, I can revert these spaces.